src/general/disallowanonymous/constraint.tmpl

```diff
@@ -24,6 +24,12 @@ spec:
               type: array
               items:
                 type: string
+            disallowAuthenticated:
+              description: >-
+                A boolean indicating whether `system:authenticated` should be
```

Comment:
Just a syntax suggestion (as this doesn't replace the other validations)

```suggestion
                A boolean indicating whether `system:authenticated` should also be
```

Reply:
Good catch ty